pkg/metainfo/kvmetainfo/buckets.go

```diff
@@ -200,8 +200,8 @@ func bucketFromMeta(ctx context.Context, bucketName string, m objects.Meta) (out
 	out.Name = bucketName
 	out.Created = m.Modified
 	// backwards compatibility for old buckets
-	out.PathCipher = storj.AESGCM
-	out.EncryptionParameters.CipherSuite = storj.EncUnspecified
+	out.PathCipher = storj.EncAESGCM
+	out.EncryptionParameters.CipherSuite = storj.EncNull
```

Comment:
since we're breaking backwards compatibility anyway, should we just remove these lines?

Reply:
make sense to me.. I removed it.